### ISSUE TITLE
Add hyper-sdk-bundle.js to npm package

### ISSUE THREAD

**Issue by basham** [RangerMauve/hyper-sdk#93]
Thanks for adding `hyper-sdk-bundle.js` to the repo, in v3.0.10. However, can you also add it to the npm package? It would make it much easier to import it to projects if it was there, instead of copying and pasting it in the project source code.

**Comment by RangerMauve** [comment]
Yeah, having prebuild bundles seems like it'd make life easier for folks.

I was thinking of using [github actions](https://github.com/datproject/sdk/pull/85#issuecomment-817349698) to publish the bundle inside the GitHub releases so people could either download or link to them directly.

Would that work for your use case? Would you be interested in editing the [GH actions](https://github.com/datproject/sdk/blob/master/.github/workflows/build.yml) to enable it?

**Comment by basham** [comment]
I've never used GitHub actions. Is that something that I can test on a fork of this repo? I could see benefit to adding it to the releases page, although I would never use it.

My hope for this issue was as simple as adding `"hyper-sdk-bundle.js"` to the [`package.json` "files" list](https://github.com/datproject/sdk/blob/v3.0.10/package.json#L47).

For my use case, I'm using [Snowpack](https://www.snowpack.dev/) as the build environment. I'd like to import the SDK from the npm package, so that it is directly included in the build process. I currently accomplish it like this:

```js
// Copy the bundle to the project.
import './hyper-sdk-bundle.js'
// Access the SDK via window.
const SDK = window.hyperSDK
```

I'd prefer something like this:

```js
import 'hyper-sdk/hyper-sdk-bundle.js'
const SDK = window.hyperSDK
```

In the long term, I'd prefer to have access to modules:

```js
import SDK from 'hyper-sdk'

// Or:

import { SDK } from 'hyper-sdk'
```

**Comment by RangerMauve** [comment]
I think the ship has sailed on this sort of approach. We don't current have web support due to changes in the hyper stack and folks should use their own bundlers.
